Vision/src/V1/ImageAnnotatorClient.php

```diff
@@ -475,6 +475,41 @@ public function webDetection($image, $optionalArgs = [])
         );
     }
 
+    /**
+     * Run object localization for an image.
+     *
+     * Example:
+     * ```
+     * $imageContent = file_get_contents('path/to/image.jpg');
+     * $response = $imageAnnotatorClient->objectLocalization($imageContent);
+     * ```
+     *
+     * @param Image $image        An image annotation request.
```

Comment:
It looks like this (and the other types for the other helpers) should be `resource|string|Image`.